_includes/tutorials/creating-first-apache-kafka-streams-application/confluent/markup/dev/run-consumer.adoc

```diff
@@ -0,0 +1,29 @@
+Now that the Kafka Streams application is running, run a command line consumer using the `ccloud` CLI to view the events.
+
+After step 4 of this tutorial, your `ccloud` context should be set to the proper environment, cluster, and API Key (see https://docs.confluent.io/ccloud-cli/current/command-reference/index.html[Confluent Cloud CLI Reference] for additional details).
+
+Then, in a new terminal window, run the following console consumer to view the events being generated by the data generator and produced to the `random-strings` topic from the `Randomizer` class in your Kafka Streams application. These are the events that have been streamed into the topology (`.stream(inputTopic, Consumed.with(stringSerde, stringSerde)`).
```

Comment:
Maybe explain this earlier in the KT, where the Util is described.

Reply:
I, personally, prefer this here. I wanted to clearly distinquish the generated events from the transformed ones, and I wanted to tell the story here by helping the user see the two event streams on the CLI.

Reply:
IMO having the user see both streams from the CLI is helpful and they can matchup what they see w/ the output of the streaming app.  I vote to keep this explanation here.